src/open_inwoner/scss/views/App.scss

```diff
@@ -255,6 +255,8 @@
 
 @font-face {
   font-family: 'Heading';
+  font-style: normal;
+  font-weight: bold;
   src: url('/media/custom_fonts/heading_font.ttf') format('truetype'),
```

Comment:
Are we going to need a separate font for bold headings, similarly to the body fonts?

Reply:
> Are we going to need a separate font for bold headings, similarly to the body fonts?

No, because by default Headings are rendered as 'bold' explicitly (just like when you'd use 'bold' for other content) so 'bold' is implied here.

Reply:
Just for my understanding: does this work because there previously was some default (numeric) value for heading `font-weight`? At first glance it doesn't make a lot of sense to see the addition of `font-weight: bold` make things less bold :sweat_smile:.

Reply:
check, thanks for the explanation :+1: